composer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "cascade-energy/distributed-locking-consul",
+  "description": "Consul implementation of the distributed locking interfaces",
+  "minimum-stability": "stable",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Charles Berube",
+      "email": "charles.berube@cascadeenergy.com"
+    }
+  ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/cberube/consul-php-sdk"
+    }
+  ],
+  "autoload": {
+    "psr-4": {
+      "CascadeEnergy\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "CascadeEnergy\\Tests\\": "tests/"
+    }
+  },
+  "require-dev": {
+    "phpunit/phpunit": "4.7.7"
+  },
+  "require": {
+    "cascade-energy/distributed-locking": "0.1.0",
+    "cascade-energy/php-exceptions": "1.0.0",
+    "sensiolabs/consul-php-sdk": "dev-x/guzzle6"
```

Comment:
So is this going to need to change to version on our fork? I gather that `dev-x/guzzle6` is just an experimental branch on their fork.

Do you want to go with this as is, or should this PR wait until our fork of the project is real.